inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/ConfigConstants.java

```diff
@@ -74,4 +74,6 @@ public class ConfigConstants {
     public static String HTTP = "http://";
     public static String HTTPS = "https://";
 
+    public static int DEFAULT_SENDER_MAX_RETRY = 3;
```

Comment:
The default value setting should remain consistent with the behavior before modification

Reply:
done